Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    /// <summary>
+    /// Part 1.
+    /// </summary>
+    public class TextTable
+    {
+        private readonly string _message;
+        private readonly int _padding;
+        private readonly int _longestWordSize;
+        private readonly char _cornerChar = '+';
+        private readonly char _horizontalBorderChar = '-';
+        private readonly char _verticalBorderChar = '|';
+        private readonly char _emptySpace = ' ';
+        private readonly string messageDivider = Environment.NewLine;
+
+        public TextTable(string message, int padding)
+        {
+            _message = message;
+            _padding = padding;
+            _longestWordSize = FindLongestWord();
+        }
+
+        public string Build()
+        {
+            if (!Test.IsStringValid(_message))
+            {
+                return string.Empty;
+            }
+
+            var result = string.Empty;
+            result += AddHorizontalBorder();
+            result += AddVerticalPadding(_padding);
+            result += AddText();
+            result += AddVerticalPadding(_padding);
+            result += AddHorizontalBorder();
+
+            return result;
+        }
+
+        private string AddText()
+        {
+            string[] words = _message.Split(messageDivider);
+            StringBuilder text = new StringBuilder();
+
+            foreach (string word in words)
+            {
+                text
+                    .Append(_verticalBorderChar)
```

Comment:
I remember Kai saying not to use StringBuilder unless I have 10+ string operations but this is what codacy requested.